control/content/directives.js

```diff
@@ -1,17 +1,38 @@
 (function (angular) {
     angular
         .module('mediaCenterContent')
-        .directive("loadImage", [function () {
+        .directive("loadImage", ["Buildfire", function (Buildfire) {
             return {
                 restrict: 'A',
                 link: function (scope, element, attrs) {
                     element.attr("src", "../../../../styles/media/holder-" + attrs.loadImage + ".gif");
-
                     var elem = $("<img>");
                     elem[0].onload = function () {
                         element.attr("src", attrs.finalSrc);
                         elem.remove();
                     };
+              
+                    attrs.$observe('finalSrc', function () {
+                        var _img = attrs.finalSrc;
+                        if (attrs.cropType && attrs.cropType == "crop") {
+                            Buildfire.imageLib.local.cropImage(_img, {
```

Comment:
this is deprecated, you should use 

https://sdk.buildfire.com/docs/imagelib/#cropimage